### ISSUE TITLE
translated_attribute_names method does not work for subclasses when using STI 

### ISSUE THREAD

**Issue by pablocrivella** [shioyama/mobility#27]
Given the following scenario:

```
class Model
  include Mobility
  translates :title
end

class SubModel < Model
end

Model.translated_attribute_names => ["title"] 
SubModel.translated_attribute_names => [] 
```

P.S. Is there already a method to get an array with the i18n_accessors? like ```["title_en", "title_es", "title_nl"]``` assuming those are the defined locales

**Comment by shioyama** [comment]
Thanks, I don't have any specs for STI so I figured things might not work entirely correctly. I'll add some specs first and then fix the issue.

Also, there is no method to get an array with i18n accessors currently. I don't think it makes sense to add such a method since you can derive the methods, but I might be convinced to change my mind about that :wink:

**Comment by pablocrivella** [comment]
The only reason i "needed" that method is because i have a model where i want to validate and save all translations them at the same time.

```ruby
# app/models/exercise_type.rb

class ExerciseType < ContentBlock
  Language.pluck(:key).map { |key| "body_#{key}" }.each do |attribute|
    validates attribute, presence: true, length: { maximum: 50 }
  end
end

# app/controllers/exercise_types_controller.rb

def exercise_type_params
  params.fetch(:exercise_type, {}).permit(:key, *translatable_fields_names)
end

def translatable_fields_names
  Language.pluck(:key).map { |key| "body_#{key}" }
end
```
But if you know of a better way it would be much appreciated :)

Thanks!

**Comment by shioyama** [comment]
@pablocrivella I pushed a fix in #30. I'll merge and release a new version sometime tomorrow.

**Comment by pablocrivella** [comment]
Awesome @shioyama 👏
